### DISCUSSION TITLE
Release Go binary that uses a shared library

### DISCUSSION THREAD

**Discussion by begelundmuller** [goreleaser/goreleaser#3303] [Q&A] [answered]
We're developing a Go tool that uses CGo to import a third-party shared library (ie. a `.so` file for Linux, `.dylib` for Mac). We have builds of the shared library available for all the platforms we target.

GoReleaser looks awesome, and we'd love to use it to sign and distribute our tool (to brew among others). But we need to sign and distribute the shared libraries along with our binary. What would be the recommended way to accomplish that using GoReleaser (if at all possible)?

**Comment by caarlos0** [comment, marked as answer]
sorry, just seen this now... I thinks might be what you want, will be released, hopefully, this weekend: https://github.com/goreleaser/goreleaser/commit/0a536f08fd4df847b3777a658aea75b788f8fa4a
